R/knitr-hooks.R

```diff
@@ -317,14 +317,43 @@ install_knitr_hooks <- function() {
           } else {
             cap_engine <- knitr_engine(options$engine)
 
+            # use logo shipped within learnr pkg (currently none)
             cap_engine_file <- system.file(file.path("internals", "icons", paste0(cap_engine, ".svg")), package = "learnr")
             if (file.exists(cap_engine_file)) {
               as.character(htmltools::div(
                 class = "tutorial_engine_icon",
                 htmltools::HTML(readLines(cap_engine_file))
               ))
             } else {
-              paste0(options$engine, " code")
+              cap_engine_val <-
+                switch(cap_engine,
+                  "bash" = "Bash",
+                  "c" = "C",
+                  "coffee" = "CoffeeScript",
+                  "cc" = "C++",
+                  "css" = "CSS",
+                  "go" = "Go",
+                  "groovy" = "Groovy",
+                  "haskell" = "Haskell",
+                  "js" = "JavaScript",
+                  "mysql" = "MySQL",
+                  "node" = "Node.js",
+                  "octave" = "Octave",
+                  "psql" = "PostgreSQL",
+                  "python" = "Python",
+                  "r" = "R",
+                  "rcpp" = "R C++",
+                  "rscript" = "Rscript",
```

Comment:
Also, to be a bit more future proof, I think [`cpp11`](https://github.com/r-lib/cpp11) should be added to the list. 

```suggestion
                  "cpp11" = "cpp11",
                  "rscript" = "Rscript",
```